diagnostic.md

```diff
@@ -2,77 +2,87 @@
 
 Place your responses inside the fenced code-blocks where indicated by comments.
 
+```comfort:4
+   clarity:4
+```
+
 What is the purpose of a backend?
 
 ```bash
-// your response here
+To store data.
 ```
```

Comment:
yes, but also secure, access and interact with that data based on user requests.